### PR TITLE
[snowflake] reuse ticdc configurations struct

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
extract configurations into a unified "config" dir, and make it easier to run test on config.

### CODE REVIEW

File: config/cdc_change_feed.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	Filter         *CfFilterCfg `json:"filter"`
+	Sink           *SinkCfg     `json:"sink"`
+	EnableOldValue bool         `json:"enable_old_value"`
+}
+
+type ChangeFeedCfg struct {
+	SinkURI    string        `json:"sink_uri"`
+	ReplicaCfg *ReplicateCfg `json:"replica_config"`
+	StartTs    uint64        `json:"start_ts"`
+}
```
Comment (Lloyd-Pottiger) [inline config/cdc_change_feed.go:30]: I think we can reuse the changefeed config from cdc

**Comment by Lloyd-Pottiger** [comment]
For better support multiple data warehouses, I have pushed a refactor, so this pr needs to rebase...

File: config/cdc_change_feed.go (hunk 1 above)
Comment (zhangjinpeng1987) [reply config/cdc_change_feed.go]: done

**Review by Lloyd-Pottiger — APPROVED** [review]
LGTM

**Pull request merged**